### PR TITLE
docs: Add note to not load balance Alert trafic

### PR DESCRIPTION
**Pull request by mxinden**
Related to issue #787

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +201,8 @@ Start your prometheus like this, for example:
 
 > Note: make sure to have a valid `prometheus.yml` in your current directory
 
+> Important: Do not load balance traffic between Prometheus and its Alertmanagers but instead point Prometheus to a list of all Alertmanagers. The Alertmanager implementation expects all Alerts to be send to all Alertmanagers to ensure high availability.
```
Comment (brancz) [inline README.md]: "Alertmanagers_,_ but"; Alerts -> alerts

Comment (mxinden) [inline README.md]: Thanks 👍

File: README.md [hunk 2]
```diff
@@ -201,6 +201,8 @@ Start your prometheus like this, for example:
 
 > Note: make sure to have a valid `prometheus.yml` in your current directory
 
+> Important: Do not load balance traffic between Prometheus and its Alertmanagers, but instead point Prometheus to a list of all Alertmanagers. The Alertmanager implementation expects all alerts to be send to all Alertmanagers to ensure high availability.
```
Comment (brancz) [inline README.md]: expects all alerts to be __sent__ to all Alertmanagers

Comment (mxinden) [inline README.md]: Sorry about that, thanks.

**Pull request merged**